### PR TITLE
Updating Threat Explorer article to include Extended Capabilities

### PR DESCRIPTION
**Pull request by vippand**

### CODE REVIEW

**Comment by yogkumgit** [comment]
@JohanFreelancer9 - kindly do the copy editor review for this PR. Thanks

**Review by MSFTTracyP — APPROVED** [review]
Update

File: microsoft-365/security/office-365-security/threat-explorer.md
Comment (JohanFreelancer9) [inline microsoft-365/security/office-365-security/threat-explorer.md]:
```suggestion
### Top targeted users
```

File: microsoft-365/security/office-365-security/threat-explorer.md
Comment (JohanFreelancer9) [inline microsoft-365/security/office-365-security/threat-explorer.md]:
```suggestion
Today we expose the list of the top targeted users in the Malware View for Emails (within the Top Malware Families section). We will be extending this view within Phish and All Email views as well, where you will be able to see the top five targeted users along with the number of attempts for each user for the corresponding view (for example, for Phish view you will be able to see the number of Phish attempts).
```

File: microsoft-365/security/office-365-security/threat-explorer.md
Comment (JohanFreelancer9) [inline microsoft-365/security/office-365-security/threat-explorer.md]:
```suggestion
You will also be able to export the list of targeted users up to a limit of 3000 along with the number of attempts for offline analysis for each email view. In addition to that, selecting No. of attempts (for example, 13 attempts below) would open a filtered view in Threat Explorer, so that you can look at more details across emails and threats for that user. 
```

File: microsoft-365/security/office-365-security/threat-explorer.md
Comment (JohanFreelancer9) [inline microsoft-365/security/office-365-security/threat-explorer.md]:
```suggestion
### Exchange transport rules
```

File: microsoft-365/security/office-365-security/threat-explorer.md
Comment (JohanFreelancer9) [inline microsoft-365/security/office-365-security/threat-explorer.md]:
```suggestion
As part of data enrichment, you should also be able to see all the different transport rules which were applied to a message. This information will be present within the Email grid view (to view this, select Column options in the grid and add Exchange Transport Rule from the Column options in the grid) as well as Details flyout in the email.
```

File: microsoft-365/security/office-365-security/threat-explorer.md
Comment (JohanFreelancer9) [inline microsoft-365/security/office-365-security/threat-explorer.md]:
```suggestion
You would be able to see both the GUID as well as the name of the transport rules which were applied to the message. Additionally, you would be able to search for the messages using the name of the transport rule. This would be a ‘Contains’ search which means you will be able to search using partial searches as well. 
```

File: microsoft-365/security/office-365-security/threat-explorer.md
Comment (JohanFreelancer9) [inline microsoft-365/security/office-365-security/threat-explorer.md]:
```suggestion
ETR search and name availability would depend on the specific role that has been assigned to you. You will need to have one of the following roles/permissions in order to view the ETR names and search.  If you do not have any of the following roles assigned to you, you will not be able to see the names of the transport rules, and search for the messages using the ETR names. However, you will be able to see the ETR label and GUID information within the Email Details. Your other experiences around viewing records in Email Grids, Email flyouts, Filters, and Export are not impacted. 
```

File: microsoft-365/security/office-365-security/threat-explorer.md
Comment (JohanFreelancer9) [inline microsoft-365/security/office-365-security/threat-explorer.md]:
```suggestion
- EXO Only - View-Only Configuration: All
```

File: microsoft-365/security/office-365-security/threat-explorer.md
Comment (JohanFreelancer9) [inline microsoft-365/security/office-365-security/threat-explorer.md]:
```suggestion
Within the email grid, Details flyout, and Exported CSV, the ETRs are presented with a Name/GUID as shown below. 
```

File: microsoft-365/security/office-365-security/threat-explorer.md
Comment (JohanFreelancer9) [inline microsoft-365/security/office-365-security/threat-explorer.md]:
```suggestion
### Inbound connectors 
```

File: microsoft-365/security/office-365-security/threat-explorer.md
Comment (JohanFreelancer9) [inline microsoft-365/security/office-365-security/threat-explorer.md]:
```suggestion
The search for connectors is ‘Contains’ in nature which means partial keyword searches should work as well. 
```

File: microsoft-365/security/office-365-security/threat-explorer.md
Comment (JohanFreelancer9) [inline microsoft-365/security/office-365-security/threat-explorer.md]:
```suggestion
Within the Main grid view, the Details flyout, and the Exported CSV, the connectors are shown in the Name/GUID format as shown below: 
```

File: microsoft-365/security/office-365-security/threat-explorer.md
Comment (JohanFreelancer9) [inline microsoft-365/security/office-365-security/threat-explorer.md]:
```suggestion
- EXO Only - Transport Rules: All
```

**Review by JohanFreelancer9 — CHANGES_REQUESTED** [review]
Copy review.

File: microsoft-365/security/office-365-security/threat-explorer.md
Comment (vippand) [reply microsoft-365/security/office-365-security/threat-explorer.md]: JohanFreelancer9 can you update these changes directly, since this allows edits by maintainers? @JohanFreelancer9

**Comment by vippand** [comment]
@JohanFreelancer9 can you please merge these changes?

**Comment by JohanFreelancer9** [comment]
@vippand Thank you for committing my edits. Unfortunately, I cannot merge the PR, it has to be done by the author/document owner.

**Review by JohanFreelancer9 — APPROVED** [review]

**Comment by vippand** [comment]
@denisebmsft Tracy/Johan have approved this PR. Unfortunately, I am unable to merge this PR, since I dont have write access to the repo. Could you help with this, please

**Comment by vippand** [comment]
Hi @MSFTTracyP / @denisebmsft can you please help with merging this PR? I dont have write access to the repo. I have merged changes suggested by @JohanFreelancer9.

**Pull request merged**
